model/project_parser.go

```diff
@@ -616,6 +628,27 @@ func retrieveFile(ctx context.Context, opts GetProjectOpts) ([]byte, error) {
 			return nil, errors.Wrap(err, "error reading project config")
 		}
 		return fileContents, nil
+	case ReadFromPatch:
+		configFile, err := thirdparty.GetGithubFile(ctx, opts.Token, opts.Ref.Owner,
```

Comment:
Can this section also be using GetFileFromPatchDiff?

Reply:
yes